src/tasks/Microsoft.NET.WebAssembly.Webcil/WebcilConverter.cs

```diff
@@ -181,33 +182,61 @@ private static void WriteHeader(Stream s, WebcilHeader header)
 
     private static void WriteSectionHeaders(Stream s, ImmutableArray<WebcilSectionHeader> sectionsHeaders)
     {
-        // FIXME: fixup endianness
-        if (!BitConverter.IsLittleEndian)
-            throw new NotImplementedException();
         foreach (var sectionHeader in sectionsHeaders)
         {
             WriteSectionHeader(s, sectionHeader);
         }
     }
 
+#if NETCOREAPP2_1_OR_GREATER
```

Comment:
Thanks. Can you try removing the `#if` if it compiles? This function might be available on all frameworks (unless I'm missing something).

Reply:
Thanks for pointing that out. I have done the same for the WriteStructure as well and hence removing the checks for net472 as well.